ios/RNKiwiMobile/RNKiwiGestureController.m

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+#import "RNKiwiGestureController.h"
+
+@implementation RNKiwiGestureController
+
+RCT_EXPORT_MODULE(RNKiwiGestureController);
+
+RCT_EXPORT_METHOD(removeGestures:(NSString *)moduleName) {
+  [[NSNotificationCenter defaultCenter] postNotificationName:@"RNKiwiRemoveGestures"
```

Comment:
Good practice - you can move this to header file as a `constant`, so that we do not use plain strings. You can see this in https://github.com/facebook/react-native/blob/master/React/Base/RCTBridge.h#L41

Look for usage of `RCTJavaScriptDidFailToLoadNotification` to see how it's defined. Also, when you define it, make sure to comment it when it's called ;)